pkg/config/load_test.go

```diff
@@ -85,6 +85,47 @@ func TestExtractRepoElementsFromPath(t *testing.T) {
 	}
 }
 
+func TestInfo_IsComplete(t *testing.T) {
+	testCases := []struct {
+		name     string
+		info     Info
+		expected bool
+	}{
+		{
+			name:     "All required members -> complete",
+			info:     Info{Org: "organization", Repo: "repository", Branch: "branch"},
+			expected: true,
+		},
+		{
+			name:     "Missing org -> incomplete",
+			info:     Info{Repo: "repository", Branch: "branch"},
+			expected: false,
+		},
+		{
+			name:     "Missing repo -> incomplete",
+			info:     Info{Org: "organization", Branch: "branch"},
+			expected: false,
+		},
+		{
+			name:     "Missing branch -> incomplete",
+			info:     Info{Org: "organization", Repo: "repository"},
+			expected: false,
+		},
+		{
+			name:     "Everything missing -> incomplete",
+			expected: false,
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			complete := tc.info.IsComplete()
+			if complete != tc.expected {
+				t.Errorf("%s: expected %t, got %t", tc.name, tc.expected, complete)
```

Comment:
Nit: There is no need to include the tests name in the error, `go test` also does this properly when using subtests (`t.Run()`)

Reply:
I'm doing this intentionally; I know the name is included but it's butchered, including it here keeps it always readable,